### ISSUE TITLE
Design the website layout

### ISSUE THREAD

**Issue by princiya** [frontendstudygroup/frontendstudygroup.github.io#1]
Come up with a design layout to be used for the website.

**Comment by Priyami** [comment]
Click on the link down below to see the model view of the website as of now. Feedbacks are much appreciated.

[https://user-images.githubusercontent.com/4775877/134753116-606f00cd-f36a-4f5e-8a56-a0354f3abd15.png](url)

**Comment by princiya** [comment]
@Priyami good one 👍 . Do you also want to create a PR with the code implementation to showcase your design?

**Comment by sunithapatel** [comment]
@Priyami This is really good, thank you for doing this! I only have one question - what goes in Products?

Other than that, everything else looks great!

**Comment by princiya** [comment]
@Priyami we have renamed `Products` to `Resources`.
I will close this issue, since there are other issues which are currently being worked in parallel.
@Priyami feel free to pick a new issue. Thanks for creating the initial mockup.
